recipes/pdf_recipe/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "diffpy.pdfmorph" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/diffpy/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 699952c747048db4a383e81fa867887a86480200a56d7a51bdd1ad43d25b392d
+
+build:
+  skip: True  # [py<3]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - matplotlib
+    - diffpy.utils
+    - scipy
+    - numpy
+    - python
```

If (and only if!) it is made `noarch` you can constrain the Python version using:
```suggestion
    - python >=3
```